### ISSUE TITLE
Vector address analyzer for LSP

### ISSUE THREAD

**Issue by seimit** [Semi-ATE/STIL#65]
Vector address (VA) analyzer for Language Server Protocol will provide vector address information of STIL file for IDE which support LSP protocol. By this way IDE can show VA number of the Vector statement for particular line in the file.
However there are some restrictions due nature of the VA calculation. For example :
- if we have pattern block used more than once in Pattern Burst block, the VA can not be shown correctly for the second execution of the same pattern block.
- VA inside Macro or Procedure bodies can not have unique number is they are called multiple times.
In summary : this functionality will work only Vectors inside Pattern blocks and only if Pattern blocks are executed only once. 
The analyzer should calculate correct VA after Macro or Procedure block execution based on number of Vector statements in the Macro or Procedure blocks, WFC passed as arguments in the Macro or Procedure blocks based on IEEE 1450.0 standard.
